src/FlipMove.js

```diff
@@ -238,7 +241,7 @@ class FlipMove extends Component {
     // we need to reset the transition, so that the NEW shuffle starts from
     // the right place.
     this.state.children.forEach((child) => {
-      const { domNode } = this.childrenData[child.key];
+      const { domNode } = this.childrenData[child.key] || {};
```

Comment:
Was this to fix an issue with Preact, or just generally a good idea to avoid issues?